css/pages/cart/cart.css

```diff
@@ -0,0 +1,100 @@
+@import url('/css/base/base.css');
+@import url('/css/base/typography.css');
+@import url('/css/base/utilities.css');
+@import url('/css/layout/main-nav.css');
+@import url('/css/pages/promos.css');
+@import url('/css/pages/featured.css');
+@import url('/css/pages/footer.css');
+@import url('/css/layout/product-nav.css');
+@import url('/css/layout/product-footer.css');
+
+.cart-management {
+    height: 100vh;
+    padding: 3rem 0;
+}
+
+.cart-main-container {
+    gap: 2rem;
+    padding: 2rem;
+}
+
+.product-system {
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
```

Comment:
for color, you could create a variable and use it

Reply:
yes i have created variables for colors its just for the box shadow and one or two as there were already many so that is why